dev-docs/INSTALLATION-AND-SETUP.md

```diff
@@ -0,0 +1,62 @@
+# Installation and Setup
+
+## Preparing Yourself
+
+To follow these instructions, it'd help if you:
+
+- have mentally prepared for experimentation and learning and/or
+- know how to open your terminal/command prompt
+- know a bit about how to navigate to different folders in your terminal
+- know a bit about installing software for your terminal, like nodejs
+
+## Github Setup
+
+If you're new to github look at [How to: fork a repo](https://help.github.com/articles/fork-a-repo/). Otherwise, fork and clone the repo, creating a link (remote branch) to the Code for Boston repo.
+
+In your termainal/command prompt, do `git checkout dev`.
+
+_Do not touch the `master` branch._
+
+## Dependency Installation
+
+### [NVM](https://github.com/creationix/nvm)
+
+```
+curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.11/install.sh | bash
```

Comment:
I added nvm here since I've found this to be extremely helpful to ensure that dev setup is consistent across machines. More than happy to hear what everyone thinks.

Reply:
I think this is a great idea.  I'd add the wget command as well, just in case they don't have curl installed.  

```
To install or update nvm, you can use the install script using cURL:

curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.11/install.sh | bash

or Wget:

wget -qO- https://raw.githubusercontent.com/creationix/nvm/v0.33.11/install.sh | bash
```